test/test_sim.py

```diff
@@ -71,6 +71,8 @@ def test_sim_3(datafiles, simulator):
         expect_string(br'INFO:  \[0\]\[dut_ut\]: tenth_test::PASSED', 'run.log')
         expect_string(br'INFO:  \[0\]\[dut_ut\]: eleventh_test::RUNNING', 'run.log')
         expect_string(br'INFO:  \[0\]\[dut_ut\]: eleventh_test::PASSED', 'run.log')
+        expect_string(br'INFO:  \[0\]\[dut_ut\]: twelfth_test::PASSED', 'run.log')
+        expect_string(br'ERROR: \[0\]\[dut_ut\]: fail_if: foo != 1 \(at .*dut_unit_test.sv line:.*\)', 'run.log')
```

Comment:
I'm confused as to why the test is passing, but an error is signaled.

Reply:
My bad, good catch
There should be `twelfth_test::FAILED`
This mistake is because of missing #113 on this branch

Reply:
Done